doc/Makefile

```
@@ -44,6 +48,18 @@ help:
 clean:
 	-rm -rf build/* source/reference/generated
 
+version-check:
+ifeq "$(GITVER)" "Unknown"
+	# @echo sdist build with unlabeled sources
+else ifneq ($(NUMPYVER),$(GITVER))
+	@echo installed numpy $(NUMPYVER) != current repo git version \'$(GITVER)\'
+	@echo use '"make dist"' or '"GITVER=$(NUMPYVER) make $(MAKECMDGOALS) ..."'
```

```suggestion
	@echo Use '"make dist"' or '"GITVER=$(NUMPYVER) make $(MAKECMDGOALS) ..."'
	@echo if you wish to make the documentation for the installed numpy. Otherwise,
	@echo ensure that the current development version is picked up, e.g. by using a
	@echo virtualenv or run from within \'python3 runtests.py --shell\'.
```
Not sure this helps much. But the error seemed a bit inviting to just copy paste the solution, when usually that is not what users want?